lib/usd/ui/layerEditor/layerTreeItem.h

```diff
@@ -138,6 +138,10 @@ class LayerTreeItem : public QStandardItem
     bool isLocked() const;
     // check if this layer is locked
     bool appearsLocked() const;
+    // is the layer system locked?
+    bool isSystemLocked() const;
+    // check if this layer is system locked
```

Comment:
I'd change the comment to reflect that this check if parents are locked and not itself. I'd even add a similar comment in the cpp to help future readers understand that not checking itself is volontary and not an oversight.